tests/search_tests.rs

```diff
@@ -48,9 +48,13 @@ fn test_identity_search() {
     let smiles_field = builder.add_text_field("smiles", STRING | STORED);
     let fingerprint_field = builder.add_bytes_field("fingerprint", FAST | STORED);
 
+    let json_options: JsonObjectOptions =
+        JsonObjectOptions::from(TEXT | STORED).set_expand_dots_enabled();
+    let _extra_data_field = builder.add_json_field("extra_data", json_options);
```

Comment:
I would just delete this line if the variable isn't used.

Reply:
So the builder is still adding the extra data field, which gets incorporated into the schema with this line. I tried deleting it and it blew up at me:)

Reply:
Ah, my bad. Just don't assign it to a variable, remove everything to the left of the word `builder`.

Reply:
Sounds good. I figured clippy might complain if I didn't assign a dummy variable, but maybe not?